packages/origin-247-certificate/src/offchain-certificate/synchronize/strategies/batch-synchronize.strategy.ts

```diff
@@ -1,37 +1,40 @@
 import { SynchronizeStrategy } from './synchronize.strategy';
-import {
-    CertificateEventRepository,
-    SynchronizableEvent
-} from '../../repositories/CertificateEvent/CertificateEvent.repository';
-import { Inject, Injectable } from '@nestjs/common';
-import { CertificateCommandRepository } from '../../repositories/CertificateCommand/CertificateCommand.repository';
-import {
-    CERTIFICATE_COMMAND_REPOSITORY,
-    CERTIFICATE_EVENT_REPOSITORY
-} from '../../repositories/repository.keys';
+import { SynchronizableEvent } from '../../repositories/CertificateEvent/CertificateEvent.repository';
+import { Injectable } from '@nestjs/common';
 import { PersistProcessor } from '../handlers/persist.handler';
 import { CertificateEventType } from '../../events/Certificate.events';
 
 @Injectable()
 export class BatchSynchronizeStrategy implements SynchronizeStrategy {
-    constructor(
-        @Inject(CERTIFICATE_COMMAND_REPOSITORY)
-        private readonly certCommandRepo: CertificateCommandRepository,
-        @Inject(CERTIFICATE_EVENT_REPOSITORY)
-        private readonly certEventRepo: CertificateEventRepository,
-        private readonly persistProcessor: PersistProcessor
-    ) {}
+    constructor(private readonly persistProcessor: PersistProcessor) {}
 
     async synchronize(events: SynchronizableEvent[]): Promise<void> {
-        const commands = await this.certCommandRepo.getByIds(events.map((e) => e.commandId));
+        const failedCertificateIds: number[] = [];
```

Comment:
Will this work? Let's discuss.

So the claim is blocking transfer, but transfer is not blocking claim, right?
It's OK for issue, but it doesn't work in case of these two

Reply:
And it is implemented only for batching but not for normal synchronization